sdk/servicebus/azure-messaging-servicebus-track2-perf/src/main/java/com/azure/messaging/servicebus/perf/processorTest.java

```diff
@@ -0,0 +1,105 @@
+package com.azure.messaging.servicebus.perf;
+
+import com.azure.core.util.CoreUtils;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.messaging.servicebus.*;
+import com.azure.messaging.servicebus.models.ServiceBusReceiveMode;
+import com.azure.perf.test.core.EventPerfTest;
+import com.azure.perf.test.core.TestDataCreationHelper;
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
+
+import java.time.Duration;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class processorTest extends EventPerfTest<ServiceBusStressOptions> {
+    private static final ClientLogger LOGGER = new ClientLogger(processorTest.class);
+
+    private static final String AZURE_SERVICE_BUS_CONNECTION_STRING = "AZURE_SERVICE_BUS_CONNECTION_STRING";
+    private static final String AZURE_SERVICEBUS_QUEUE_NAME = "AZURE_SERVICEBUS_QUEUE_NAME";
+
+    private final String CONNECTION_STRING;
+    private final String QUEUE_NAME;
+
+    private final AtomicInteger total = new AtomicInteger();
+
+    /**
+     * Creates an instance of performance test.
+     *
+     * @param options the options configured for the test.
+     * @throws IllegalStateException if SSL context cannot be created.
+     */
+    public processorTest(ServiceBusStressOptions options) {
+        super(options);
+
+        CONNECTION_STRING = System.getenv(AZURE_SERVICE_BUS_CONNECTION_STRING);
+        if (CoreUtils.isNullOrEmpty(CONNECTION_STRING)) {
+            throw LOGGER.logExceptionAsError(new IllegalArgumentException(
+                String.format("Environment variable %s must be set", AZURE_SERVICE_BUS_CONNECTION_STRING)));
+        }
+
+        QUEUE_NAME = System.getenv(AZURE_SERVICEBUS_QUEUE_NAME);
+        if (CoreUtils.isNullOrEmpty(QUEUE_NAME)) {
+            throw LOGGER.logExceptionAsError(new IllegalArgumentException(
+                String.format("Environment variable %s must be set", AZURE_SERVICEBUS_QUEUE_NAME)));
+        }
+
+    }
+
+    @Override
+    public Mono<Void> setupAsync() {
+        ServiceBusSenderClient sender = new ServiceBusClientBuilder()
+            .connectionString(CONNECTION_STRING)
+            .sender()
+            .queueName(QUEUE_NAME)
+            .buildClient();
+
+        for (int i = 0; i < options.getMessagesToSend(); i++) {
```

Comment:
what if I specify 1,000,000 messages.
We should split the input with the max messages the batch send can handle.